### PR TITLE
DE-18299: Added renderer for file upload component

### PR DESCRIPTION
**Pull request by sushil-meher-nice**

### CODE REVIEW

**Comment by sushil-meher-nice** [comment]
PHPStan check is failed. Need https://github.com/BrandEmbassy/ui-components/pull/284 changes in order to pass.

File: src/FormField/FileUpload/FileUploadFieldRenderer.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+use BrandEmbassy\Components\NetteForm\FormField\FieldRenderer;
+use BrandEmbassy\Components\NetteForm\OptionField;
+use BrandEmbassy\Components\UiComponent;
+use Nette\ComponentModel\IComponent;
+use Nette\Forms\Controls\UploadControl;
+use function assert;
+
+final class FileUploadFieldRenderer implements FieldRenderer
+{
+    public function render(IComponent $control, string $accept = '', bool $isMultiple = false): UiComponent
```
Comment (michalbrauner) [inline src/FormField/FileUpload/FileUploadFieldRenderer.php:15]:
```suggestion
    public function render(IComponent $control, string $acceptAttributeValue= '', bool $isMultiple = false): UiComponent
```

**Review by michalbrauner — COMMENTED** [review]
I would do one renaming

File: src/FormField/FileUpload/FileUploadFieldRenderer.php (hunk 1 above)
Comment (sushil-meher-nice) [reply src/FormField/FileUpload/FileUploadFieldRenderer.php]: done.

**Review by michalbrauner — APPROVED** [review]

**Review by roshan-deshmukh-nice — APPROVED** [review]

**Pull request merged**